wrapanapi/systems/redfish.py

```diff
@@ -35,3 +51,84 @@ def info(self):
 
     def __del__(self):
         """Disconnect from the API when the object is deleted"""
+
+    def server_stats(self, *requested_stats, **kwargs):
+        # Get the requester which represents the class of this method's caller
+        requester = kwargs.get('requester')
```

Comment:
Nitpick: last three lines can be simplified to `def server_stats(self, requester=None, *requested_stats, **_)`. It would be even better to require `requester` as a first, non-optional parameter, but that would mean breaking the API and fixing all the sites that call this method.

Reply:
Did you try to run it and it failed? Because looking at the method call, things should work as I described. In any case, this was just a nitpick, so feel free to ignore it.

Reply:
Sorry, my bad. You are right, suggested refactoring will probably error-out because `requester` parameter will have multiple values assigned. There is no sane way of handling this under python 2.

Reply:
My approach is even more radical: changing the call directly:

https://github.com/ManageIQ/integration_tests/pull/7735
https://github.com/ManageIQ/wrapanapi/pull/331